lib/pf/cmd/pf/import/nodes.pm

```diff
@@ -15,24 +15,93 @@ use strict;
 use warnings;
 use base qw(pf::cmd);
 use Role::Tiny::With;
+use pf::constants qw($TRUE $FALSE);
 use pf::constants::exit_code qw($EXIT_SUCCESS);
+use pf::file_paths qw($install_dir);
 with 'pf::cmd::roles::show_parent_help';
 
+=head2 parseArgs
+
+Parse the arguments for this command
+
+=cut
+
 sub parseArgs {
     my ($self) = @_;
-    my ($file) = $self->args;
-    if($file) {
-        $self->{file} = $file;
-        return 1;
+
+    my ( $file, @params ) = $self->args;
+
+    unless(defined($file)) {
+        print STDERR "You must specify a file name\n";
+        return $FALSE;
+    }
+
+    unless(-f $file) {
+        print STDERR "File $file doesn't exist...\n";
+        return $FALSE;
     }
-    return 0;
+
+    require pf::config;
+
+    my %params = (
+        'columns'           => "mac",
+        'default-role'      => "default",
+        'default-unregdate' => "2038-01-01 00:00:00",
+        'default-voip'      => "no",
+        'default-owner'     => "default",
+        'delimiter'         => "comma",
+    );
+    foreach my $param ( @params ) {
+        my @data = split('=', $param);
```

Comment:
The following would be more clear
```perl
my ($name, $val) = split('=', $param);
```

Reply:
This was your code I copy/pasted from another file BTW ;) but I'll fix it

Reply:
Which file so we can improve readability there also.

Reply:
Its in lib/pf/cmd/pf/ipmachistory.pm

I wouldn't "pollute" this PR with code cleanup in case we use it again for patching a customer so we could do it in devel in a separate task